mdata/reorder_buffer.go

```diff
@@ -79,3 +79,22 @@ func (rob *ReorderBuffer) Get() []schema.Point {
 
 	return res
 }
+
+func (rob *ReorderBuffer) Reset() {
+	for i := range rob.buf {
+		rob.buf[i].Ts = 0
+		rob.buf[i].Val = 0
+	}
+	rob.newest = 0
+}
+
+func (rob *ReorderBuffer) Flush() []schema.Point {
+	res := rob.Get()
+	rob.Reset()
+
+	return res
+}
+
+func (rob *ReorderBuffer) HasData() bool {
```

Comment:
super minor comment, but [empty](https://github.com/search?l=Go&q=func+empty&type=Code&utf8=%E2%9C%93) seems a bit more common than [hasdata](https://github.com/search?l=Go&q=func+hasdata&type=Code&utf8=%E2%9C%93) in Go source code. plus allows us to write positive checks instead of negations

Reply:
k, i'll update & then merge